### ISSUE TITLE
couple of small issues

### ISSUE THREAD

**Issue by sre** [radare/radare2#6394]
Hi,

The Debian build infrastructure [0] found a few smaller issues with the source/buildsystem. If nobody beats me to it, I can prepare patches for them after CCC.

```
I pointer-cast-size-mismatch cmd_zign.c:354 (armel, armhf, hppa, hurd-i386, i386, kfreebsd-i386, m68k, mips, mipsel, powerpc, powerpcspe, sh4, x32)
I pointer-cast-size-mismatch cmd_zign.c:369 (armel, armhf, hppa, hurd-i386, i386, kfreebsd-i386, m68k, mips, mipsel, powerpc, powerpcspe, sh4, x32)
I pointer-cast-size-mismatch cmd_zign.c:47 (armel, armhf, hppa, hurd-i386, i386, kfreebsd-i386, m68k, mips, mipsel, powerpc, powerpcspe, sh4, x32)
W implicit-declaration p/debug_native.c:1492 (armel, armhf)
W implicit-declaration run.c:247 (hurd-i386, kfreebsd-i386)
W implicit-declaration run.c:540 (hurd-i386, kfreebsd-i386)
W implicit-declaration run.c:594 (arm64, armel, armhf, hppa, hurd-i386, i386, kfreebsd-i386, m68k, mips, mips64el, mipsel, powerpc, powerpcspe, ppc64el, s390x, sh4, sparc64, x32)
```

[0] https://qa.debian.org/bls/packages/r/radare2.html

-- Sebastian

**Comment by radare** [comment]
please update this errors, because doesnt match current source lines

**Comment by dequis** [comment]
#6398 fixed most of this

Second and third warnings were fixed by 8f323094916a221648f9c12814baa027f203d3b4

The line in the first warning was removed as part of the rewrite in 3f89eec3b0e926122602a10c2422825a1d7d4e6e and there are no equivalent calls anywhere else.

Close?
